.github/workflows/release-pipeline.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: Release docker image
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+jobs:
+  release-docker:
+    name: Release docker image
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Deploy docker image
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: grzesuav/metacontroller
```

Comment:
will this be changed to docker hub repo i.e. `metacontrollerio`
i.e. https://hub.docker.com/orgs/metacontrollerio

Reply:
yes, I used my own to test the flow